index.html

```diff
@@ -133,5 +144,47 @@ <h2>About Me</h2>
     </div>
     </section>
     <br>
+    <section class="footer">
+    <h2 class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in <span class="toll">touch.</span></h2>
+    
+    <form  class="player" action="https://formspree.io/f/xayagqrn" method="post">
+    <ul class="name">
+    <li class="form">
+    <label for="fullName">
+    <input type="text" name="full-name" placeholder="  Full name" maxlength="30" id="fullName" required>
+        </label></li>
+        
+    <li class="form">
+    <label for="email">
+        <input type="email" name="email" id="email" placeholder="  Email address" required>
```

Comment:
- [ ] The border bottom on these two inputs is taking too much space, please check this 😉 

Figma  | Yours 
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/145396959-e75e8809-e78f-4465-9584-d2cbd0153d81.png) | ![image](https://user-images.githubusercontent.com/77038610/145396817-fcba113a-1b37-47f7-a059-f581ef0f81af.png)

Reply:
I have modified the bottom bottom of the inputs in the contact form to look like the Figma template. Thanks for your kind review